recipes/kat/2.0.8/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -x -e
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+export BOOST_INCLUDE_DIR=${PREFIX}/include
+export BOOST_LIBRARY_DIR=${PREFIX}/lib
+export LIBS='-lboost_system -lboost_program_options -lboost_filesystem -lboost_timer'
+
+export CXXFLAGS="-DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export LDFLAGS="-L${BOOST_LIBRARY_DIR} -lboost_filesystem -lboost_system"
+
+# fix automake
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/aclocal
```

Comment:
@bgruening , you were too quick for me! I was just getting ready to test this. ;-) In any case it shouldn't hurt, right?